### PR TITLE
add material component to Detail Module

### PR DESCRIPTION
**Pull request by dweitz43**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
branch update

* **What is the current behavior?** (You can also link to an open issue here)
Detail module does not include material components

* **What is the new behavior (if this is a feature change)?**
implement material component in Detail Module, update to master branch as well

* **Other information**:

### CODE REVIEW

File: src/app/+detail/index.ts [hunk 1]
```diff
@@ -20,8 +20,9 @@ export const routes = [
     BrowserModule,
     FormsModule,
     RouterModule.forChild(routes),
+    MdModule.forRoot()
```
Comment (zigzag95) [inline src/app/+detail/index.ts]: You don't need to use .forRoot() on MdModule since md providers are already loaded by app

File: tsconfig.json [hunk 2]
```diff
@@ -17,7 +17,6 @@
       "es6"
     ],
     "types": [
-      "core-js",
```
Comment (zigzag95) [inline tsconfig.json]: I don't get any conflict and removing this I get errors Promise Unknown

File: src/app/+detail/index.ts (hunk 1 above)
Comment (dweitz43) [inline src/app/+detail/index.ts]: @zigzag95 i wasnt sure about that for a lazily loaded module, thanks for the tip! updating the pr now...

File: tsconfig.json (hunk 2 above)
Comment (dweitz43) [inline tsconfig.json]: @zigzag95 i think this was a result of the merge with master, as i believe it is not in tsconfig.json in the master branch....however i didnt get any errors removing it, but i am adding it back now

**Comment by dweitz43** [comment]
@gdi2290 seems like the build fails because of a problem installing node-sass, but i dont experience this issue at all... i wonder if this can still be merged? or if i should just wait until the repo is upgraded to rc.6 now that it has been released?

**Comment by gdi2290** [comment]
@katallaxie I avoid any modules that might cause problems when installing for example node-sass and CompressionPlugin

**Pull request closed without merging**